src/Microsoft.Sbom.Targets/GenerateSbomTask.cs

```diff
@@ -163,4 +159,24 @@ public override bool Execute()
             return false;
         }
     }
+
+    /// <summary>
+    /// Checks the user's input for Verbosity and assigns the
+    /// associated EventLevel value for logging.
+    /// </summary>
+    private EventLevel ValidateAndAssignVerbosity()
+    {
+        if (string.IsNullOrEmpty(this.Verbosity))
+        {
+            Log.LogMessage("No verbosity level specified. Setting verbosity level at \"LogAlways\"");
```

Comment:
You could use the expression `EventLevel.LogAlways` in the log message. Something like this:
```suggestion
            Log.LogMessage($"No verbosity level specified. Setting verbosity level at \"{EventLevel.LogAlways}\"");
```

Reply:
Updated!